components/landing/TheQuickStart/ProvidersList.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <section class="providers">
+    <h4>Providers</h4>
+    <ul class="providers__list">
+      <li v-for="item in providersList" :key="item.id">
+        <a class="providers__list__item" :href="item.link" target="_blank">
+          {{ item.title }}
+          <span class="providers__list__cta">
+            <span class="providers__list__cta-label">Learn more</span>
+            <Launch16 class="providers__list__cta-icon" />
+          </span>
+        </a>
+      </li>
+    </ul>
+    <AppCta
+      class="providers__btn"
+      v-bind="providersPage"
+    />
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+import { GeneralLink } from '~/constants/appLinks'
+
+type ProviderLink = {
```

Comment:
Why you cannot use the `GeneralLink` here instead of creating a new type?

Reply:
Done in https://github.com/Qiskit/qiskit.org/pull/3020/commits/7c7909727e57817c961742721e2c985d837d63b3